src/Components/Announcement.vue

```diff
@@ -209,6 +223,17 @@ export default {
 			)
 		},
 
+		isScheduled() {
+			return this.scheduleTime && this.scheduleTime !== null
+		},
+
+		scheduledLabel() {
+			if (this.scheduleTime) {
+				return t('announcementcenter', 'scheduled at {time}', { time: this.scheduleDateFormat })
+			}
+			return 'BUG:' + this.scheduleTime
```

Comment:
?

Reply:
This was just there for debugging, I'll remove it :+1: